doc/radosgw/pubsub-module.rst

```diff
@@ -11,8 +11,8 @@ events. Events are published into predefined topics. Topics can be subscribed to
 can be pulled from them. Events need to be acked. Also, events will expire and disappear
 after a period of time. 
 
-A push notification mechanism exists too, currently supporting HTTP and
-AMQP0.9.1 endpoints, on top of storing the events in Ceph. If events should only be pushed to an endpoint
+A push notification mechanism exists too, currently supporting HTTP,
+AMQP0.9.1 and Kafka endpoints, on top of storing the events in Ceph. If events should only be pushed to an endpoint
```

Comment:
"on top of storing the events in Ceph"--this clause is a bit confusing, though the second part (use bucket notifications if..." clears it up.

Reply:
will rephrase